packages/@icon-magic/cli/README.md

```diff
@@ -0,0 +1,96 @@
+# @icon-magic/cli
+
+> Icon Magic command line.
+
+## Install
+
+Using npm:
+
+```sh
+npm install --save-dev @icon-magic/cli
+```
+
+or using yarn:
+
+```sh
+yarn add @icon-magic/cli --dev
+```
+
+## Glossary
+
+- **Asset**: A single file containing a logo/image. Assets can be of multiple
+  types .svg, .png, .webp
+- **Variant**: a version of the icon that has its own underlying path elements.
+  Variants are always of .svg type and form the input of the entire icon build
+  process.
+- **Icon**: The class representing a group of variants that belong to the same
+  icon. For example, two variants of the home icon can be a filled version and
+  an outline version of the same home icon. Within the file system, an icon is a
+  directory that consists of all the variant assets and a corresponding
+  iconrc.json config file.
+- **iconrc.(js|json)** A config file for a single icon or a group of icons with
+  paths to the various icon directories and their variants at minimum. The
+  config further be caustomaized
+- **Flavor**: An asset obtained after applying build/generate plugins on the
+  source .svg file. A flavor will also contain assets for each type, i. e.,
+  paths to the .svg, .png and .webp version of that flavor.
+
+## Usage
+
+For a detailed list of the terminology, refer to the glossary above.
+
+```
+Usage:
+  icon-magic [command] [options] <directories ...>
+
+Commands:
+  build               Construct flavors that an icon from its variants, after applying the build plugins.
+  generate            Generates the flavors of the icon in the extension types that it can be consumed.
+  distribute          Moves an icon from the source to the destination, applying plugins if specified.
+
+Options:
+  -c, --config        Path to config file name
+  -i, --inputPath     Path to the input directory of icons
+  -o, --outputPath    Path to the output directory where the generated assets are to be written to
+  -h, --help          Display usage
+  -v, --version       Display version
```

Comment:
Minor nitpick: You'll need to add `program.version('1.2.3');` in order for commander to output version.

https://www.npmjs.com/package/commander#version-option